changelog/fix_unused_render_content_documentation.md

```diff
@@ -0,0 +1 @@
+* [#1116](https://github.com/rubocop/rubocop-rails/issues/1116): Update `Rails/UnusedRenderContent` documentation to recommend use of `head` instead of `render`. ([@samrjenkins][])
```

Comment:
Can you remove this changelog entry? Because the cop behavior remains the same.

Reply:
sure!

Reply:
I have amended the commit